tox.ini

```diff
@@ -97,6 +97,7 @@ commands =
     lint: pylint --rcfile=tests/pyproject.toml tests
     lint: pydocstyle lms tests bin
     lint: pycodestyle lms tests bin
+    ruff: ruff check lms bin
```

Comment:
I see ruff is not checking the tests. Was that just to keep this PR smaller or is there another reason?